pkgs/applications/audio/soundtracker/default.nix

```diff
@@ -0,0 +1,49 @@
+{ stdenv
+, fetchurl
+, pkg-config
+, autoconf
+, gtk2
+, alsaLib
+, SDL
+, jack2
+, goocanvas # graphical envelope editing
+}:
+
+with stdenv.lib; stdenv.mkDerivation rec {
```

Comment:
```suggestion
stdenv.mkDerivation rec {
```

Reply:
Did you test this? I'm using `optional` from `stdenv.lib`

Reply:
Tested without this change. This should work (untested):

```suggestion
let inherit (stdenv.lib) optionals; in

stdenv.mkDerivation rec {
```

Reply:
In the end I just qualified `optional`, since I'm only using it one time